azkaban-common/build.gradle

```diff
@@ -72,6 +72,13 @@ dependencies {
   testRuntime('com.h2database:h2:1.4.193')
 }
 
+task copySqlFiles(type: Copy) {
+  from project(':azkaban-sql').file('src/sql')
+  into file('build/resources/test/sql2')
```

Comment:
Please use "$buildDir". The code already has examples for this. Hardcoding the build directory sometimes creates weird problems. For example, the last time when the web server had certain code failing because some `moment.js` was hardcoded to copy to 'build' directory?